service/src/main/kotlin/fi/espoo/evaka/emailclient/EmailConfig.kt

```diff
@@ -8,13 +8,17 @@ import com.amazonaws.services.simpleemail.AmazonSimpleEmailService
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.core.env.Environment
+import org.springframework.core.env.getRequiredProperty
 
 @Configuration
 class EmailConfig {
     @Bean
     fun emailClient(client: AmazonSimpleEmailService, env: Environment): IEmailClient =
         when (env.getProperty("application.email.enabled", Boolean::class.java, false)) {
-            true -> EmailClient(client = client)
+            true -> EmailClient(
+                client = client,
+                whitelist = env.getRequiredProperty<List<String>>("application.email.whitelist").map { Regex(it) }
```

Comment:
As this is now a required property (when mail is enabled), should we have a default for it in `application.yaml`? Or if the idea is to force an active decision for the person deploying this service -> we should inform Trevaka about this one.

Reply:
It shouldn't be a required property, I just forgot to change that method. Good catch 👍